cluster.go

```diff
@@ -102,7 +102,8 @@ type ClusterConfig struct {
 	// Initial keyspace. Optional.
 	Keyspace string
 
-	// Number of connections per host.
+	// The size of connection pool for each host. The Pool will be filled during the first request execution.
```

Comment:
I'm not sure if this is 100% correct, if `InitialHostLookup`  is enabled (it's the default) then the pools for the discovered hosts will be filled during session initialization

Reply:
You are right!
I double-checked it with the timeout, and it turned out that the pool filled during initialization. When I tested it last time the pool had not been filled before the query execution because [pool filling runs in separate gorutine](https://github.com/OleksiienkoMykyta/gocql/blob/974fa1211ccef10a418966fd7b00984a39ed9232/session.go#L292) and it took more time to finish than the query execution.  
Thank you for noticing the issue! 
I can replace it with something like 
```  
The size of the connection pool for each host. The pool filling runs in separate gourutine during the session initialization phase. 
Also, it describes a maximum number of connections at the same time.
Notice: There is no guarantee that pool filling will be finished in the initialization phase.
Default: 2 
```
What do you think?

Reply:
Hmm interesting, did you use `SingleHostReadyPolicy`? If you don't use this policy then it should wait until all hosts have at least 1 connection in the pool before the session initialization is done.

Check [this](https://github.com/OleksiienkoMykyta/gocql/blob/974fa1211ccef10a418966fd7b00984a39ed9232/session.go#L326-L330) and [this](https://github.com/apache/cassandra-gocql-driver/blob/7b7e6affbfddf36e9cbc1068e96ecb9c7cc0af18/connectionpool.go#L454-L482).

Considering this, it would probably be good to mention that gocql will always try to get 1 connection on each host pool during session initialization AND it will attempt to fill each pool afterwards asynchronously if `NumConns > 1`

Reply:
What about something like that? 
```
// The size of the connection pool for each host.
// The pool filling runs in separate gourutine during the session initialization phase.
// gocql will always try to get 1 connection on each host pool
// during session initialization AND it will attempt
// to fill each pool afterward asynchronously if NumConns > 1.
// Notice: There is no guarantee that pool filling will be finished in the initialization phase.
// Also, it describes a maximum number of connections at the same time.
```
I have added `gocql will always try to get 1 connection on each host pool during session initialization AND it will attempt to fill each pool afterward asynchronously if NumConns > 1` as you suggested, so there is no more room for unclearness.